src/ape/managers/chain.py

```diff
@@ -403,6 +407,11 @@ def _contract_types_cache(self) -> Path:
         network_name = self._network.name.replace("-fork", "")
         return self._network.ecosystem.data_folder / network_name / "contract_types"
 
+    # I don't love this. I think I'd rather it be at the config level.
+    @property
+    def _contracts_mapping_cache(self) -> Path:
+        return self._network.ecosystem.data_folder / "contracts_map.json"
```

Comment:
```suggestion
        return self._network.ecosystem.data_folder / "deployments.json"
```

I am tryin to think of way to lessen the confusion between this feature and the contracts caching we do.